packages/client/src/user/userReducer.ts

```diff
@@ -414,6 +421,15 @@ export const userFormReducer: LoopReducer<IUserFormState, UserFormAction> = (
             failActionCreator: submitFail
           }
         )
+      ]
+      if (isSelfUpdate) {
+        commandList.push(
+          Cmd.action(modifyUserDetails({ mobile: userDetails.mobile }))
```

Comment:
What if something else other than mobile no is updated?

Reply:
I think other than the mobile number no value changes on the settings page!

Reply:
Ok then it's fine